src/main/java/net/neoforged/neoforge/fluids/capability/IFluidHandler.java

```diff
@@ -6,14 +6,21 @@
 package net.neoforged.neoforge.fluids.capability;
 
 import net.neoforged.neoforge.fluids.FluidStack;
-import net.neoforged.neoforge.fluids.IFluidTank;
+import net.neoforged.neoforge.transfer.ResourceHandlerUtil;
+import net.neoforged.neoforge.transfer.handlers.resources.IResourceHandler;
+import net.neoforged.neoforge.transfer.resources.FluidResource;
+import net.neoforged.neoforge.transfer.resources.IResource;
+import net.neoforged.neoforge.transfer.transaction.TransactionContext;
+import org.jetbrains.annotations.ApiStatus;
 
 /**
- * Implement this interface as a capability which should handle fluids, generally storing them in
- * one or more internal {@link IFluidTank} objects.
- * <p>
- * A reference implementation is provided {@link TileFluidHandler}.
+ * Implement this interface as a capability which should handle fluids
+ * 
+ * @deprecated Use {@link IResourceHandler} bound by type {@link FluidResource} instead.
```

Comment:
ideally these could use the format `@deprecated Replace with {@link foo}` and IDEA will actually process that (whether it works due to the interface change is another question)